### PR TITLE
Add labels file

### PR DESCRIPTION
**Pull request by dbkr**
Copied from the old matrix-org react-sdk labels (which weren't synced from github it seems).

## Checklist

-   [ ] Tests written for new code (and old code if feasible).
-   [ ] New or updated `public`/`exported` symbols have accurate [TSDoc](https://tsdoc.org/) documentation.
-   [ ] Linter and other CI checks pass.
-   [ ] I have licensed the changes to Element by completing the [Contributor License Agreement (CLA)](https://cla-assistant.io/element-hq/matrix-react-sdk)

### CODE REVIEW

File: .github/labels.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+- name: "T-Defect"
+  description: "Bugs, crashes, hangs, vulnerabilities, or other reported problems"
+  color: "#98e6ae"
+- name: "T-Deprecation"
+  description: "A pull request that makes something deprecated"
+  color: "#98e6ae"
+- name: "T-Enhancement"
+  description: "New features, changes in functionality, performance boosts, user-facing improvements"
+  color: "#98e6ae"
+- name: "T-Task"
```
Comment (langleyd) [inline .github/labels.yml:21]: and this

**Comment by florianduros** [comment]
Can't we use the labels from element-meta too?

File: .github/labels.yml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  color: "#5319E7"
+- name: "backport staging"
+  description: "Label to automatically backport PR to staging branch"
+  color: "#B60205"
+- name: "Dependencies"
+  description: "Pull requests that update a dependency file"
+  color: "#0366d6"
+- name: "Sponsored"
+  color: "#b506d8"
+- name: "T-Defect"
```
[the remark this answers is not in the document]
Comment (dbkr) [reply .github/labels.yml:12]: oh right... I can't find where though? https://github.com/element-hq/element-meta/blob/develop/.github/labels.yml is empty...

[the remark this answers is not in the document]
Comment (dbkr) [reply .github/labels.yml:12]: Oh! I thought they all came from yaml definitions - okay, gotcha.

**Review by florianduros — APPROVED** [review]

**Pull request merged**